actions/release/1.0/internal/pkg/diceyml.go

```diff
@@ -54,9 +54,11 @@ func fillDiceYml(cfg *conf.Conf, storage *StorageURL) (string, error) {
 		return "", err
 	}
 
-	//假如是service模式，需要将service中的cmd塞入dice.yaml中的cmd中
+	//假如是service模式，需要将service中的cmd塞入dice.yaml中的cmd中；并判定是否使用的镜像修改成retag中的镜像
 	if cfg.Services != nil {
 		insertCommands(&dCopy, cfg)
+		// 默认将制品中使用的镜像修改成retag中的镜像
```

Comment:
Do not impact the existing functionality; the logic for `useRetagImage` should not take effect if the user has not provided `RetagImage`.

Reply:
Added `RetagImage == ""` judgment logic in the `useRetagImage` function